python/cudf/cudf/_fuzz_testing/orc.py

```diff
@@ -67,6 +67,19 @@ def generate_input(self):
             dtypes_meta, num_rows, num_cols = _generate_rand_meta(
                 self, dtypes_list
             )
+            if num_cols == 0:
+                """
+                If a dataframe has no columns, then pyorc writer will throw
```

Comment:
I wondered what the desired behavior is here, i.e. whether ORC as a format supports having no columns.
I think we also have some issues writing empty dataframes to ORC.

Reply:
I guess we need to write an `empty struct`. With pyorc we can do it this way:

```python
>>> import pyorc
>>> import pandas as pd
>>> output = open("sample.orc", "wb")
>>> writer = pyorc.Writer(output, pyorc.Struct())
>>> writer.close()
>>> pd.read_orc('sample.orc')
Empty DataFrame
Columns: []
Index: []
```

Looks like I also need to make some code-changes in this PR. I'll update this PR.